src/config/config.ts

```diff
@@ -38,8 +38,8 @@ const config = convict({
   env: {
     doc: "The application environment.",
     env: "NODE_ENV",
-    format: ["LOCAL_DEV", "DEV", "test", "prod", "staging"],
-    default: "LOCAL_DEV",
+    format: ["dev", "test", "prod", "staging"],
```

Comment:
could we use this opportunity to change "test" to "test-dev" / "local-test" instead?

Reply:
do you feel like we should add a `vapt` env here or should we do that separately in another PR that configures stuff for vapt

Reply:
`test` is actually set by jest itself: see https://jestjs.io/docs/environment-variables

I think it makes sense to allow the vapt entry as well, have added it in https://github.com/isomerpages/isomercms-backend/pull/659/commits/0134f9647031bffc67dd09f61cb84bc738e9f811